### PR TITLE
Removing Housing evals

### PR DESCRIPTION
**Pull request by ajgajg1134**
Check one:
- [x] Semantic Change: something about the meaning of the text is different
- [ ] Non-semantic Change: Spelling, grammar, or formatting changes.

Summary of change(s):

(See title)

### CODE REVIEW

File: bylaws.tex [hunk 1]
```diff
@@ -257,24 +257,8 @@
...
-The participating members may then choose to award one, two, or three housing points to each member.
-Points are awarded on the basis of contributions to House.
-Above average members receive three points, average members two points, and below average members one point.
-\\* \\*
-A quorum of two-thirds of the Total Number of Possible Votes is required for the evaluation to be valid.
-Neither Absentee nor Proxy votes are allowed.
+\bsubsection{Housing Points}
+All members may notify the evaluations director before the end of the academic year that they would like to receive two housing points for use in the next academic year.
 \\* \\*
 All votes during the Housing Evaluation Process must be Closed Ballot, Three Tiered Vote as defined in \ref{Three Tiered}.
```
Comment (liam-middlebrook) [inline bylaws.tex]: This should be removed

**Comment by liam-middlebrook** [comment]
Line 252 needs to be re-worded away from the text 'The first process, known as the Housing Evaluation,'

**Comment by liam-middlebrook** [comment]
@ajgajg1134 Can you fix the commit with what I mentioned above?

**Comment by ajgajg1134** [comment]
@liam-middlebrook Should be fixed, This whole section should likely be re-written to improve readability and clarity, but I believe that's beyond the scope of this PR (and my capabilities without some help)

**Comment by liam-middlebrook** [comment]
LGTM :shipit:

In the future we should strictly adhere to posting a publicly viewable diff before an amendment comes to vote.

**Pull request merged**